### PR TITLE
CONFIG: RAIL-1448 Update of translations

### PR DESCRIPTION
**Pull request by sitina**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-app-components:
- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3119/ : SUCCESS in 9m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3867/ : SUCCESS in 14s

File: src/translations/ja-JP.json [hunk 1]
```diff
@@ -55,10 +55,15 @@
...
     "visualizations.measure.arithmetic.change.title": "{secondMeasureTitle} から {firstMeasureTitle} の変化",
-    "visualizations.legend.text.top": "上： ",
-    "visualizations.legend.text.bottom": "下： ",
-    "visualizations.legend.text.left": "左： ",
-    "visualizations.legend.text.right": "右： ",
+    "visualizations.legend.text.top": "上",
+    "visualizations.legend.text.bottom": "下",
+    "visualizations.legend.text.left": "左",
+    "visualizations.legend.text.right": "右",
+    "visualizations.legend.text.colon": ":",
```
Comment (nadtan) [inline src/translations/ja-JP.json]: Do we need a space after `:` cause in the previous version `"visualizations.legend.text.right": "右： "` has a space after the text?

Comment (sitina) [reply src/translations/ja-JP.json]: I believe yes, will update that

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3134/ : SUCCESS in 9m 38s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3884/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1668/ : SUCCESS in 39m 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3136/ : SUCCESS in 9m 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/855/ : SUCCESS in 30s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/862/ : FAILURE in 32m 24s

**Comment by sitina** [comment]
@petrjanu shall we try to re-merge or is there any fix needed in order to make this pass? Thank you

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1675/ : SUCCESS in 40m 23s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3149/ : SUCCESS in 12m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/859/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/871/ : SUCCESS in 28m 32s

**Pull request merged**